src/server/game/Entities/GameObject/GameObject.cpp

```diff
@@ -1757,6 +1757,8 @@ void GameObject::Use(Unit* user)
         return;
     }
 
+    sOutdoorPvPMgr->HandleCustomSpell(user->ToPlayer(), spellId, this);
```

Comment:
please call this only if user->ToPlayer() doesn't return NULL, some handlers might expect this parameter never to be NULL